std/range/package.d

```diff
@@ -3877,7 +3894,6 @@ pure unittest
     assert(b == [ "c", "b", "a" ]);
```

Comment:
I suppose `t1, t2` might be clearer than `c, d` as they are tuples. Also I think UFCS style would be better: `zip(...).sort!(...)`.